.gitignore

```diff
@@ -6,3 +6,17 @@ test_db.sqlite3
 
 .tox
 .cache
+
+.idea/
+.ipynb_checkpoints/
+next.filmfest.by.iml
```

Comment:
I just added idea special files to git ignore. And couple more specific files like ipython notebook ones